triton/examples/whisper/whisper.rst

```diff
@@ -0,0 +1,141 @@
+Whisper
+==========
+
+.. highlight:: console
+
+This uses :doc:`Singularity containers </triton/usage/singularity>`,
+so you should refer to that page first for general information.
+
+There are two variants of Whisper available. The "standard" Whisper uses 
+whisper-ctranslate2, which is a CLI for faster-whisper, a reimplementation 
+of OpenAI's Whisper using Ctranslate2. Original repository for this 
+project can be found 
+`here <https://github.com/Softcatala/whisper-ctranslate2>`__.
+
+The second variant is whisper-diarization, which is a fork of faster-whisper 
+with support for speaker detection (diarization). 
+Original repository for this project can be found 
+`here <https://github.com/MahmoudAshraf97/whisper-diarization>`__.
+
+Of these two, whisper-diarization runs noticable slower and has less versatile 
+options. Using base Whisper is recommended if speaker detection is not necessary.
+
+Usage (Whisper)
+-------------------
+
+This example shows you a sample script to run Whisper.
+
+.. code-block:: bash
+
+    module load whisper
+    srun --mem=4G singularity_wrapper run your_audio_file.wav --model_directory $medium_en --local_files_only True --language en
+
+Option ``--model_directory $medium_en`` Tells whisper to use a local model, in 
+this case the model ``medium.en`` with the path to the model given through 
+the environment variable ``$medium_en``. For list of all local models, you can 
+run ``echo $model_names`` as long as the module is loaded. You can also give it 
```

Comment:
```suggestion
run ``echo $model_names`` as long as the module is loaded. (These models are pre-downloaded by us and the variables
are defined when the module is loaded.)
You can also give it 
```
I wonder if this is obvious, worth saying, or worth saying but is unnecessary information?